### PR TITLE
Update Anime Planet

### PR DESCRIPTION
**Pull request by SlowLife1661**
image
image
image
image
image

### CODE REVIEW

File: websites/A/Anime Planet/presence.ts [hunk 1]
```diff
@@ -243,28 +285,16 @@ presence.on("UpdateData", async () => {
...
 
-    if (!buttonsE) delete presenceData.buttons;
-
     presence.setActivity(presenceData);
 });
 
-interface EHref extends HTMLElement {
+interface ElementHref extends HTMLElement {
   href: string
 }
```
Comment (ririxidev) [inline websites/A/Anime Planet/presence.ts:299]: unecessary kinda. just do ``(document.querySelector('h2.sub > a') as HTMLElement).href``

File: websites/A/Anime Planet/presence.ts [hunk 2]
```diff
@@ -49,6 +56,15 @@ presence.on("UpdateData", async () => {
       strings = getStrings();
     }
 
+    if (!oldHref){
+      oldHref = document.location.href;
+      presence.hideSetting(["AnimeDetails", "AnimeState", "MangaDetails", "MangaState"]);
```
Comment (ririxidev) [inline websites/A/Anime Planet/presence.ts:61]: Why you hide settings in this time. I don't understand it.

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @SlowLife1661!
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/A/Anime Planet/presence.ts [hunk 3]
```diff
@@ -214,7 +246,7 @@ presence.on("UpdateData", async () => {
           },
           {
             label: (await strings).viewSeries,
-            url: document.querySelector<EHref>('h2.sub > a').href
+            url: document.querySelector<ElementHref>('h2.sub > a').href
           }
```
Comment (Bas950) [inline websites/A/Anime Planet/presence.ts:250]: HTMLElement should work

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by SlowLife1661** [comment]
Now I broke it again LOL I will refork

**Pull request closed without merging**